frontend_tests/zjsunit/namespace.js

```diff
@@ -30,20 +71,12 @@ function load(request, parent, isMain) {
         return obj;
     }
 
-    if (filename.endsWith(".hbs") && !filename.includes("frontend_tests/node_tests")) {
+    if (filename.endsWith(".hbs") && filename.includes(TEMPLATE_PATH)) {
```

Comment:
Should this be its own commit?  It'd be nice to have a commit message explaining why this logic change is correct.

Reply:
I amended the commit message.